bucket/ht-rust.json

```
@@ -0,0 +1,15 @@
+{
+    "homepage": "https://github.com/ducaale/ht/",
+    "description": "Yet another HTTPie clone in Rust.",
+    "license": "MIT",
+    "version": "0.5.0",
+    "url": "https://github.com/ducaale/ht/releases/download/v0.5.0/ht-x86_64-pc-windows-msvc.zip",
+    "hash": "eca94991b75ac4cb718f6c1449ae521ea692c73ab5ee1ff326e28636f3314c1a",
+    "bin": "ht.exe",
+    "checkver": {
+        "github": "https://github.com/ducaale/ht"
+    },
```

When homepage is same as Github repo link, there is no need to specify twice

```suggestion
    "checkver": "github",
```